src/lisp/kernel/clasp-builder.lsp

```diff
@@ -343,19 +377,19 @@ Return files."
                 done
                   ))
              (if (>= wpid 0)
-                 (let* ((finished-entry-triplet (gethash wpid jobs))
-                        (finished-entry (first finished-entry-triplet))
-                        (finished-job-counter (second finished-entry-triplet))
-                        (result-stream (third finished-entry-triplet)))
-                   (finished-one finished-entry finished-job-counter wpid result-stream)
-                   (when reload (reload-one finished-entry))
+                 (let* ((finished-entries-triplet (gethash wpid jobs))
+                        (finished-entries (first finished-entries-triplet))
+                        (finished-counter (second finished-entries-triplet))
+                        (result-stream (third finished-entries-triplet)))
+                   (finished-some finished-entries finished-counter wpid result-stream)
+                   (when reload (reload-some finished-entries))
                    (decf child-count))
                  (error "wait returned ~d  status ~d~%" wpid status)))
-           (when entry
+           (when entries
              (multiple-value-bind (maybe-error pid-or-error result-stream)
                  (core:fork t)
                (if maybe-error
-                   (error "Could not fork when trying to build ~a" entry)
+                   (error "Could not fork when trying to build ~a" entries)
```

Comment:
Is this correct? I am not sure what ~a does. entries is a list.

Reply:
Yes - it's correct. The ~a means "print argument in an Aesthetically pleasing way".